tools/smarter_device_manager/BUILD

```diff
@@ -1,8 +1,19 @@
 load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+load("@io_bazel_rules_docker//container:container.bzl", "container_push")
 
 go_image(
     name = "smarter_device_manager_image",
     binary = "@com_gitlab_arm_research_smarter_device_manager//:smarter-device-manager",
+		base = "@buildbuddy_go_image_base//image",
```

Comment:
does this need formatting?

Reply:
lol, you're fast. yeah, i need to run buildfix real quick.